test/terraform_scp_example_test.go

```diff
@@ -0,0 +1,325 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"io/ioutil"
+
+	"path/filepath"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/gruntwork-io/terratest/modules/ssh"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformScpExample(t *testing.T) {
+	t.Parallel()
+
+	//os.Setenv("SKIP_setup", "true")
+	//os.Setenv("SKIP_validate_file", "true")
+	//os.Setenv("SKIP_validate_dir", "true")
+	//os.Setenv("SKIP_validate_asg", "true")
+	//os.Setenv("SKIP_teardown", "true")
```

Comment:
scrub ;)

Reply:
Got rid of it - I wish there was a way to leave it in safely. Alas...

Reply:
👍